charts/tensorleap/templates/rabbitmq-sts.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: rabbitmq
+  labels:
+    app: rabbitmq
+spec:
+  serviceName: rabbitmq
+  replicas: 1
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: rabbitmq
+  template:
+    metadata:
+      labels:
+        app: rabbitmq
+    spec:
+      containers:
+        - name: rabbitmq
+          image: rabbitmq:3.9.22
+          imagePullPolicy: IfNotPresent
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                  - rabbitmqctl
+                  - stop_app
+          env:
+            - name: RABBITMQ_ERL_COOKIE
+              valueFrom:
+                configMapKeyRef:
+                  name: rabbitmq-config
+                  key: erlangCookie
+            - name: RABBITMQ_DATA_DIR
+              value: /var/lib/rabbitmq
+            - name: RABBITMQ_MNESIA_BASE
+              value: /var/lib/rabbitmq/mnesia
+            - name: RABBITMQ_LOG_BASE
+              value: /var/log/rabbitmq
```

Comment:
See if you can move those to the config file please

Reply:
Sorry, this isn't supported